src/styles/stopwatch/stopwatch.scss

```diff
@@ -0,0 +1,41 @@
+.main {
+  @include horizontal-center-flex;
+
+  min-height: 100vh;
+
+  margin: 0;
+}
+
+.stopwatch {
+  @include horizontal-center-flex;
```

Comment:
your 'horizontal-center-flex' contains the margin but it's not allowed to use margins for BEM blocks